oneview/resource_server_profile_template.go

```diff
@@ -1216,13 +1211,15 @@ func resourceServerProfileTemplateCreate(d *schema.ResourceData, meta interface{
 	config := meta.(*Config)
 
 	serverProfileTemplate := ov.ServerProfile{
-		Name:               d.Get("name").(string),
-		Type:               d.Get("type").(string),
-		Affinity:           d.Get("affinity").(string),
-		SerialNumberType:   d.Get("serial_number_type").(string),
-		WWNType:            d.Get("wwn_type").(string),
-		MACType:            d.Get("mac_type").(string),
-		HideUnusedFlexNics: d.Get("hide_unused_flex_nics").(bool),
+		Name:                     d.Get("name").(string),
+		Type:                     d.Get("type").(string),
+		Affinity:                 d.Get("affinity").(string),
+		SerialNumberType:         d.Get("serial_number_type").(string),
+		WWNType:                  d.Get("wwn_type").(string),
+		MACType:                  d.Get("mac_type").(string),
+		HideUnusedFlexNics:       d.Get("hide_unused_flex_nics").(bool),
+		Description:              d.Get("description").(string),
```

Comment:
i can see in schema description is set as only optional but server_profile_description set as both optional and computed any reason?

Reply:
Fixed